cmake/projects/odb-pgsql/hunter.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2015, Ruslan Baratov, Alexandre Pretyman
+# All rights reserved.
+
+if(DEFINED HUNTER_CMAKE_PROJECTS_ODB-PGSQL_HUNTER_CMAKE)
+  return()
+else()
+  set(HUNTER_CMAKE_PROJECTS_ODB-PGSQL_HUNTER_CMAKE 1)
+endif()
+
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_package(odb)
+hunter_add_package(PostgreSQL)
```

Comment:
Ok, I used the `odb-sqlite` template, where in the past we've decided to put dependencies in `projects/../hunter.cmake` and on the scheme script. I'll remove these.